<launch>
    <node pkg="robot_driver" type="robot_driver_odom" name="robot_driver_odom" output="screen">
        <param name="odom_frame" value="odom"/>
        <param name="base_frame" value="base_link"/>
        <param name="odom_topic" value="/odom"/>
        <param name="serial_port" value="/dev/ttyUSB1"/>
        <param name="freq" value="10"/>
    </node>
</launch>